README.md

```diff
@@ -13,6 +13,8 @@ Configuration
 - `b:phpfold_heredocs = 1` - Fold HEREDOCs and NOWDOCs.
 - `b:phpfold_docblocks = 1` - Fold DocBlocks.
 - `b:phpfold_doc_with_funcs = 1` - Fold DocBlocks. Overrides `b:phpfold_docblocks`.
+- `b:phpfold_curlies = 1` - Folds within `{}`.
+- `b:phpfold_misc = 1` - Fold square and normal brackets `()` and `[]`
```

Comment:
I think it'd more explicit what the option does if it were named `b:phpfold_brackets`.  Thoughts?